ComfortZoneApp/app/src/main/java/com/example/comfortzone/GetLocationCallback.java

```diff
@@ -0,0 +1,5 @@
+package com.example.comfortzone;
+
+public interface GetLocationCallback {
+    void location(String lat, String lon);
```

Comment:
You can rename this callback function to onLocationUpdated or something similar.